.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Pull Request
+about: Open a Pull Request for this project
+title: ''
+labels: 'type: pull request'
+assignees: ''
+
+---
+
+## Description
+
+Fixes issue: ___
```

Comment:
```suggestion
<!-- Link an issue to this PR. If none, this line can be removed -->
Fixes ___
```
Based on [GitHub docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)